scaleway/resource_instance_volume.go

```diff
@@ -238,18 +238,20 @@ func resourceScalewayInstanceVolumeDelete(ctx context.Context, d *schema.Resourc
 	}
 
 	err = resource.Retry(d.Timeout(schema.TimeoutDelete), func() *resource.RetryError {
-		volumeResp, err := instanceAPI.GetVolume(&instance.GetVolumeRequest{
```

Comment:
Can you add the new way to [wait](https://github.com/scaleway/terraform-provider-scaleway/blob/master/scaleway/helpers_baremetal.go#L121) with the default timeouts